pkg/kubelet/runonce.go

```diff
@@ -91,8 +91,8 @@ func (kl *Kubelet) runOnce(pods []*v1.Pod, retryDelay time.Duration) (results []
 		res := <-ch
 		results = append(results, res)
 		if res.Err != nil {
-			// TODO(proppy): report which containers failed the pod.
-			glog.Infof("failed to start pod %q: %v", format.Pod(res.Pod), res.Err)
+			failcontainername, _ := kl.getFailContainer(res.Pod)
+			glog.Infof("failed to start container %v for pod %q: %v", failcontainername, format.Pod(res.Pod), res.Err)
```

Comment:
Nit: capitalize `Failed` while you're at it.

Reply:
Done